src/Api/Controllers/AppsController.php

```diff
@@ -36,13 +50,37 @@ class AppsController extends BaseController
      */
     public function onConstruct()
     {
+        $random = new Random();
         $this->model = new Apps();
+        $this->model->key = $random->uuid();
+        $this->model->is_actived = 1;
         $this->additionalSearchFields = [
             ['is_deleted', ':', '0'],
             ['id', ':', implode('|', $this->userData->getAssociatedApps())],
         ];
     }
 
+    /**
+     * Process the create request and records the object.
+     *
+     * @return ModelInterface
+     *
+     * @throws Exception
+     */
+    protected function processCreate(RequestInterface $request) : ModelInterface
+    {
+        //process the input
+        $request = $this->processInput($request->getPostData());
+
+        if (array_key_exists('settings', $request)) {
+            $this->model->setSettings(json_decode($request['settings'], true));
```

Comment:
@rwhite27 you just need to do `$app->set('key', 'value')` remember this is a hash table